railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -116,12 +117,12 @@ def config
     def config_when_updating
```

Comment:
Does this need to handle both the old and new paths?

Reply:
That's something I was wondering. If these files were edited to contain things others than config sets, they shouldn't be moved.